src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -239,7 +239,15 @@ public static class PublishedContentExtensions
             // Check if the culture has been set, otherwise use from Umbraco.
             if (culture == null)
             {
-                culture = UmbracoContext.Current.PublishedContentRequest.Culture;
+                if (UmbracoContext.Current.PublishedContentRequest != null)
```

Comment:
We'd probably need to check if `UmbracoContext.Current != null` too - otherwise may throw null reference.